src/bootstrap.js

```diff
@@ -162,9 +162,28 @@ function startup({webExtension}, reason) {
     }
   }
 
+  try {
+    Services.telemetry.registerScalars(TELEMETRY_CATEGORY, {
+      "datastoreCount": {
+        kind: Services.telemetry.SCALAR_TYPE_COUNT,
+        keyed: false,
+        record_on_release: false,
+        expired: false,
+      },
+    });
+  } catch (e) {
+    if (e.message === "Attempt to register scalar that is already registered.") {
+      // eslint-disable-next-line no-console
+      console.log("telemetry scalar already registered; skipping registration");
+    } else {
+      throw e;
+    }
+  }
+
   webExtension.startup().then(({browser}) => {
     Services.telemetry.recordEvent(TELEMETRY_CATEGORY, "startup",
                                    "webextension");
+    // Services.telemetry.scalarSet(TELEMETRY_CATEGORY + ".datastoreCount", 50);
```

Comment:
is there a need for keeping this comment?  If not, we should remove it so we don't have "dead code".